src/main/kotlin/no/nav/tilleggsstonader/sak/behandling/vent/SettPåVentBeskrivelseUtil.kt

```diff
@@ -13,26 +13,30 @@ object SettPåVentBeskrivelseUtil {
         oppgave: Oppgave,
         frist: LocalDate,
         tidspunkt: LocalDateTime = LocalDateTime.now(),
+        kommentar: String?,
```

Comment:
Dette var tidligere en ønske om at det ikke trengte å være med.
Så har vi det problemet med at kommentaren kan ha behov for sensitiv informasjon, noe som ikke skal vises i beskrivelsen på oppgaven

Reply:
Ja, vet at det var bevisst at vi ikke gjorde dette tidligere. Saksbehandlerne bruker foreløpig gosys, og det har vært flere oppgaver som som ved feil har blitt plukket fordi saksbehandler som bruker gosys ikke har fått med seg at oppgaven en på vent. De er ikke vant til å bruke mapper, og derfor har de egentlig ignorert det feltet.

Derfor har det vært et ønske om også oppdatere gosys-oppgaven. Skjønner at sensitiv info kan være litt skummelt (er ikke dette også et problem i EF), så kanskje man enten må tydeliggjøre at gosys-oppgaven oppdateres. Evt. kan man velge å ikke oppdatere beskrivelsen med den faktiske kommentaren, men heller bare helle til "Oppgave satt på vent" eller noe sånt 🤔

Reply:
Ja muligens man kan ha en "tittel" (første setning) som er at oppgaven er på vent, så fjerner man den raden når man tar oppgaven av vent?

Reply:
Vet ikke om man trenger å fjerne at oppgaven er satt på vent-innslaget i beskrivelsen?

Reply:
Det beror litt på hvordan man løser det. 
Hvis det er en tekst i begynnelsen av beskrivelsen så kan det kanskje være det? 

Samtidig vet jeg ikke om "på-vent-oppgaver" er de eneste som er problemet her? 
Er ikke vanlige oppgaver som ikke er på vent også et problem? Sånn generellt burde vel aldri oppgaver av typen "Behandle sak" aldri bli plukket i gosys

Reply:
Nei, de burde ikke det. Men det har blitt gjort flere ganger så virker ikke som de ser på feltet? Mulig det mest en en tilvenningssak. Men egentlig enig i at det hadde vært greit å ikke oppdatere oppgavebeskrivelsen, men kanskje bare legge inn "Satt på vent av..."